installing/installing_bare_metal_ipi/ipi-install-prerequisites.adoc

```diff
@@ -21,6 +21,8 @@ Before starting an installer-provisioned installation of {product-title}, ensure
 
 include::modules/ipi-install-node-requirements.adoc[leveloffset=+1]
 
+include::modules/installation-minimum-resource-requirements.adoc[leveloffset=+1]
```

Comment:
Small nit. The module says `<3> As with all user-provisioned installations,`. 

It's not a big deal but you might want to remove this clause.

Reply:
It needed an ipi-bare-metal context so that the table reflects CPU instead of vCPU, and also removed RHEL 8.6 for compute notes as IPI installs RHCOS. Also removed the third footnote as a result.